magento/magento2
# MC-38993: Magento\Authorization\Model\ResourceModel\Rules missing update method

Added deprecated tag to the method. Will be deleted in next major release





### Description (*)

Method was never implemented and used. Migrated from Magento 1

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#29128

### Manual testing scenarios (*)

1. Create a module that will use method \Magento\Authorization\Model\Rules::update
2. Run the code
Expected result: 
An exception that the method is obsolete but not a fatal error
3. Run Static Tests
Expected result: 
An exception that method can not be used as it is deprecated 
 
### Questions or comments


### Contribution checklist (*)
